posts/insitudiff/index.qmd

```diff
@@ -0,0 +1,42 @@
+---
+title: "InSituDiff: perturbation analysis for spatial transcriptomics"
+author:
+  - name: Patrick Danaher
+    orcid: 0000-0002-2844-5883
+    affiliations:
+      - ref: nstg
+      - ref: patrickjdanaher
+date: "2024-10-29"
+date-modified: "2024-10-29"
+categories: [exploratory analysis]
+draft: false
+image: figures/insitudiff-tile.png
+---
+
+
+# Background 
+
+A typical study design will contrast several disease or treated samples with one or more controls. 
+In these studies, it can be difficult to identify the (interesting) ways in which the disease samples
+are perturbed compared to the controls. 
+To this end, we have created an R package, "InSituDiff", containing a suite of tools
+ for exploring how cellular neighborhoods are perturbed in disease compared to controls. 
+
+The heart of InSituDiff is the measurement of how "cellular neighborhoods" (a target cell and its nearest neighbors) contrast with their *most similar control cellular neighborhood*. 
+This analysis produces a perturbation score for all cells * genes.
+With these perturbation scores in hand, numerous useful analyses become possible:
+
+1. Spatial maps of gene perturbation scores are often more informative than maps of expression level.
+2 We can identify highly perturbed genes, and we can quantify the spatial dependence of their perturbations.
```

Comment:
This line isn't rendering as a separate line. I think you need a period after the 2 to get it to make a new line: 
![{9026F3B0-C87E-45D2-972B-39F064EDAE53}](https://github.com/user-attachments/assets/7330dab1-6921-4b42-b213-03f842ce131b)